src/gui/mainwindow.cpp

```diff
@@ -187,6 +189,16 @@ void MainWindow::neovimMaximized(bool set)
 	}
 }
 
+void MainWindow::neovimFrameless(bool set)
+{
+	setWindowFlag(Qt::FramelessWindowHint, set);
+	// DD: Need call show o make the widget visible again.
+	// https://doc.qt.io/qt-5/qwidget.html#windowFlags-prop
+	show();
+	// DD: It seems there is no event representing the change of flags
+	m_nvim->api0()->vim_set_var("GuiWindowFrameless", set ? 1 : 0);
```

Comment:
Let's avoid the `foo ? 1 :0` ternary (if it is not too much effort).

You might be able to leverage `variant_not_zero(set)`

Reply:
Here and elsewhere

Reply:
`vim_set_var(str, bool)` generates `v:bool` in vimscript while the variable and function accept integers (I did it the same way as `GuiWindowMazimized` and others).